src/Data/test/ilResultTests.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+use namespace ILIAS\Data;
+
+/**
+ * Tests working with result object
+ *
+ * @author Stefan Hecken <stefan.hecken@concepts-and-training.de>
+ *
+ * @version 1.0.0
+ */
+class ilResultTests extends PHPUnit_Framework_TestCase {
```

Comment:
Kein `il`-Prefixe mehr in `src`, bitte. Du kannst das imho einfach `ResultTest` nennen.